test/server/cards/02_SHD/upgrades/MandalorianArmor.spec.ts

```diff
@@ -0,0 +1,37 @@
+describe('Mandalorian Armor', function() {
+    integration(function(contextRef) {
+        describe('Mandalorian Armor\'s attached ability', function() {
+            it('If attached unit is a Mandalorian, give a Shield token to it, or dont', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['ketsu-onyo#old-friend', 'the-mandalorian#wherever-i-go-he-goes', 'battlefield-marine'],
```

Comment:
Can we have a vehicle unit in play so we can confirm that it doesn't appear in the target list with `toBeAbleToSelectExactly` (which is already set up with the correct list of targets anyway since the vehicle shouldn't be included). That will confirm that the attachment filter is working.